qiskit_bot.yaml

```diff
@@ -836,6 +836,7 @@ notifications:
     - "@jhwatrous"
   "learning/courses/foundations-of-quantum-error-correction/fault-tolerant-quantum-computing/threshold-theorem":
     - "@jhwatrous"
+  "learning/courses/quantum-diagonalization-algorithms/exam":
```

Comment:
No one will get a notification with this syntax. Each file needs the names listed